src/index.ts

```diff
@@ -122,6 +123,24 @@ module.exports = (app: ChartProviderApp): Plugin => {
               description:
                 'Map URL (for tilelayer include {z}, {x} and {y} parameters, e.g. "http://example.org/{z}/{x}/{y}.png")'
             },
+            cached: {
+              type: 'boolean',
+              title: 'Cache tiles locally',
+              description:
+                'If enabled, tiles fetched from the server will be cached locally to reduce bandwidth usage.',
+              default: false
+            },
+            headers: {
+              type: 'array',
+              title: 'Headers',
+              description:
+                'List of http headers to be sent to the server when requesting map tiles.',
```

Comment:
Maybe _when requesting map tiles_ so that the connection to the caching proxy functionality is obvious.